### PR TITLE
Fix parameters in docstrings

### PR DESCRIPTION
**Pull request by SvoONs**
Maybe this is not the most pressing issue, but while browsing the sphinx API docs I noticed that the parameters in docstrings are mostly malformatted.
snapshot-docs

This pull request will fix the formatting for `dabl/plot/utils.py`. If desired I can do it also for the remaining files in this PR.

### CODE REVIEW

**Comment by SvoONs** [comment]
This is what the docs would look like after merge:
snapshot-docs-fixed

**Comment by amueller** [comment]
Thanks! I think it's good to fix the formatting. I'm not entirely sure this is the right fix, though. Numpydoc used to require the extra space, I'm not sure that has changed.
But the rendering after the fix also doesn't look right, it should be more similar to imbalanced-learn:

https://imbalanced-learn.readthedocs.io/en/stable/generated/imblearn.under_sampling.ClusterCentroids.html#imblearn.under_sampling.ClusterCentroids

I think there's something not right with the configuration of numpdoc.

**Comment by amueller** [comment]
I tried to unpin the numpydoc version in #199 but that didn't do it. There must be some other misconfiguration or wrong syntax somewhere.

**Comment by amueller** [comment]
@glemaitre in case you have a minute, any chance you know what's wrong with how we use numpydoc in dabl?

**Comment by amueller** [comment]
The parameters are a table in imblearn but not in dabl, so there's something different in the html structure already... huh...

**Comment by glemaitre** [comment]
The that you can have is due to the rtd template: https://github.com/scikit-learn-contrib/imbalanced-learn/blob/master/doc/conf.py#L320

**Comment by amueller** [comment]
Sorry accidental close

**Comment by amueller** [comment]
Hm you're right. How does the CSS change the html though? Well I'm not gonna question that.
Though then you get weird centering behavior in the api docs:
image

**Comment by amueller** [comment]
Somehow ``basic.css`` doesn't show up in the imbalance-learn html but it does in the dabl html and messes things up.... http://contrib.scikit-learn.org/metric-learn/metric_learn.html

**Comment by glemaitre** [comment]
Yes I decided that centring was OK until somebody want to fix it :)

**Comment by amueller** [comment]
metric learn looks correct but I have no idea how:
http://contrib.scikit-learn.org/metric-learn/generated/metric_learn.LFDA.html#metric_learn.LFDA

**Comment by amueller** [comment]
I opened an issue on sklearn-project-template: https://github.com/scikit-learn-contrib/project-template/issues/58

**Comment by amueller** [comment]
```
# Switch to old behavior with html4, for a good display of references,
# as described in https://github.com/sphinx-doc/sphinx/issues/6705
html4_writer = True
```
is the magic switch...

**Comment by amueller** [comment]
Maybe not the most elegant fix but seems to work: #200
